Tutorials/Making_Maps_with_Cartopy.ipynb

```diff
@@ -18,21 +18,14 @@
    "cell_type": "code",
```

Comment:
<p>Maybe explain what/where/why <code>geolon_t</code> and <code>geolat_t</code> are masked?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/431/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>done</p>